core-bundle/src/Controller/AbstractController.php

```diff
@@ -19,7 +19,7 @@
 
 abstract class AbstractController extends SymfonyAbstractController implements ServiceAnnotationInterface
 {
-    public static function getSubscribedServices()
+    public static function getSubscribedServices(): array
```

Comment:
Isn’t this a BC break?

Reply:
Hm, I guess you are right. If we added the return type, every child class has to add it as well. (Which is already the case for all our child classes.)

When did we add the abstract controller?

Reply:
In 4.9 (see https://github.com/contao/contao/pull/603).

Reply:
We should have added the return type there. @contao/developers I assume we cannot add it as a bugfix in version 4.9.3, can we?

Reply:
I would consider this as BC break.